perfdash/downloader.go

```diff
@@ -106,23 +107,23 @@ func (j *JobToCategoryData) ServeCategoryNames(res http.ResponseWriter, req *htt
 func (j *JobToCategoryData) ServeMetricNames(res http.ResponseWriter, req *http.Request) {
 	jobname, ok := getURLParam(req, "jobname")
 	if !ok {
-		fmt.Printf("Url Param 'jobname' is missing\n")
+		klog.Infof("Url Param 'jobname' is missing\n")
```

Comment:
Don't these logs mean that the request returns nothing? This is something like 404 Not Found case, maybe we should log such issues in this method (and also in `ServeMetricNames` below) as Warning?